src/Codeception/Lib/Driver/Db.php

```diff
@@ -215,7 +215,12 @@ protected function generateWhereClause(array &$criteria)
         $params = [];
         foreach ($criteria as $k => $v) {
             if ($v === null) {
-                $params[] = $this->getQuotedName($k) . " IS NULL ";
+                if (stripos($k, ' !=') > 0) {
+                    $params[] = $this->getQuotedName(str_ireplace(" !=", '', $k)) . " IS NOT NULL ";
```

Comment:
`strpos` and `str_replace` works well enough for `!=` because there is only one way to write it.

Reply:
Whoops, force of habit. I'll fix it up.